model-optimizer/mo/front/caffe/extractor.py

```diff
@@ -75,27 +75,20 @@ def common_caffe_fields(node: Node) -> dict:
 def caffe_extractor(node: Node, lowered_keys_map: dict) -> (bool, dict):
     if node.has_valid('op') and node.op == 'Identity':
         return True, {}
+
     result = common_caffe_fields(node)
     supported = False
-    name = None
 
-    layer_type = result['type'].lower()
-    if layer_type in lowered_keys_map:
-        layer_type = lowered_keys_map[layer_type]
+    op = result['op'].lower()
```

Comment:
Why it was changed from `type` to `op`? How did you test that this change doesn't break anything?

Reply:
reverted